models/ideas.js

```diff
@@ -14,6 +15,18 @@ function createIdea(header, description, askedSum, userId) {
     return newUser;
 }
 
+function updateIdea(ideaId, idea) {
+    let ideas = getIdeas();
+    const idea = idea;
+
+    const updateIdeas = ideas.reduse((updateIdeas, idea) => {
+        if (idea.id === ideaId) {
+            return [updateIdeas, updateIdea]
```

Comment:
There's an issue with this line - can you spot it?

Reply:
updateIdeas not coppied? [...updateIdeas, updateIdea] ?

Reply:
Yes